packages/plans-grid/src/plans-table/style.scss

```diff
@@ -25,7 +25,6 @@
 	}
 
 	// in this variation, ticks are black-ish
-	.plans-feature-list__item svg path,
```

Comment:
Probably related to this other comment of mine, but why are we deleting this line?

Reply:
It's indeed related to [your other comment](https://github.com/Automattic/wp-calypso/pull/47796#discussion_r531588492). When all ticks have to be dark gray, there's no need for this selector anymore.